docs/src/index.md

```diff
@@ -10,6 +10,7 @@ To get started, please read the [Tutorial](@ref) page.
 
 !!! tip "Latest news: Agents.jl v4.3"
       * New functions `dump_to_csv` and `populate_from_csv!` to save and load agent data to and from CSV files
+      * New functions `save_checkpoint` and `load_checkpoint` to save and load entire models to and from files

```

Comment:
Since this is new functionality outside of what we've already implemented, we will have to release 4.4. So this section needs to be updated accordingly.